lib/adal.js

```diff
@@ -189,6 +200,28 @@ var AuthenticationContext = (function () {
         // callback from redirected page will receive fragment. It needs to call oauth2Callback
     };
 
+    AuthenticationContext.prototype.loginPopup = function (urlNavigate) {
+        var popupWindow = window.open(urlNavigate, "login", 'width=483, height=600');
```

Comment:
Should we try to position this somewhere other than upper left? It would be nice to center/middle it.